pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,36 @@
+include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the customer portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged-in.
+* The module to be published is uploaded.
+
+.Procedure
+
+. Navigate to link:https://pantheon.cee.redhat.com/#/search[].
```

Comment:
This is an internal URL; we might want to change this to a replaceable such as <pantheonurl> or an attribute for now.

Reply:
changed to a placeholder attribute